assignments/arrays.js

```diff
@@ -63,34 +63,59 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+for(car in inventory){
+  if(inventory[car]["id"] === 33){
+    console.log(`Car ${inventory[car]["id"]} is a ${inventory[car]['car_year']} ${inventory[car]['car_make']} ${inventory[car]['car_model']}`)
+  }
+}
+
 
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory.length;
+for(car in inventory){
+  if(inventory[car]["id"] === lastCar){
+    console.log(`The last car is a ${inventory[car]['car_make']} ${inventory[car]['car_model']}`)
+  }
+}
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+for (car in inventory){
```

Comment:
Based on the repeated use of for in I am gonna assume that you know how to write a basic for loop however I would have liked you to have written it out once. For in has some use cases where it will not work and you are going to have to use a basic loop. Also am not sure it was necessary to use ```toLowerCase()```. Maybe you could convince me if we actually discussed it 😃